### PR TITLE
test new helpers

### PR DESCRIPTION
**Pull request by maurolepore**
Hold on ... I thinks I need to cleanup commits.

### CODE REVIEW

File: R/utils-tests.R [hunk 1]
```diff
@@ -74,9 +74,12 @@ year_range <- function(p) {
 }
 
 abort_if_year_range_is_5yr_already <- function(data) {
+  # This is an assertion inside a test
+  # nocov start
   if (diff(range(data$year)) == 5) {
     stop("The default year range must not be 5.", call. = FALSE)
   }
+  # nocov end
```
Comment (maurolepore) [inline R/utils-tests.R:82]: Avoid being penalized by test-coverage

File: R/to_title.R [hunk 2]
```diff
@@ -65,6 +65,6 @@ spell_out_technology <- function(x) {
   label <- to_title(x)
   label <- sub("^(?i)ice", "ICE", label)
   label <- sub("cap$", " Capacity", label)
-  label <- sub("_hdv$", "Heavy Duty Vehicles", label)
+  label <- sub("Hdv$", "Heavy Duty Vehicles", label)
```
Comment (maurolepore) [inline R/to_title.R:68]: This was a bug. At this point "_" is gone and "hdv" became "Hdv"

File: tests/testthat/test-to_title.R [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  # styler: off
+  data <- tribble(
+              ~input,          ~output,
+          "a.string",       "A String",
+    "another_string", "Another String",
+          "b.STRING",       "B String"
+  )
+  # styler: on
+  helper <- to_title
+  expect_equal(helper(data$input), data$output)
```
Comment (maurolepore) [inline tests/testthat/test-to_title.R:11]: There are more compact ways to structure these tests but I think this one is a particularly clear one.

File: tests/testthat/test-to_title.R [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  # styler: off
+  data <- tribble(
+           ~input,     ~output,
+      "projected", "Portfolio",
+        "target_",  "Scenario",
+  "anything else", "Benchmark",
+      # Edge cases
+      "Projected", "Benchmark",
+       ".target_", "Benchmark",
+            "xyz", "Benchmark"
```
Comment (maurolepore) [inline tests/testthat/test-to_title.R:39]: This is to characterize edge behavior that might be considered a bug.

File: tests/testthat/test-to_title.R [hunk 5]
```diff
@@ -0,0 +1,58 @@
+test_that("to_title() converts labels correctly", {
+  # styler: off
+  data <- tribble(
+              ~input,          ~output,
+          "a.string",       "A String",
+    "another_string", "Another String",
+          "b.STRING",       "B String"
+  )
+  # styler: on
```
Comment (maurolepore) [inline tests/testthat/test-to_title.R:9]: Turn off styler to avoid destroying the clear allignment -- which is the whole point of structuring the tests this way.

**Comment by maurolepore** [comment]
Favour #438
